BackofficeBundle/Resources/public/coffee/dataTable/DataTableChannel.coffee

```diff
@@ -0,0 +1,11 @@
+###*
+* @channel DataTable
+* Event available
+*  - clearCache(tableId) clearCache of DataTable
+*  - draw(tableId)
+###
+(($, OpenOrchestra, DataTable) ->
+  DataTable.Channel = new (Backbone.Wreqr.EventAggregator)
+) jQuery,
+  window.OpenOrchestra = window.OpenOrchestra or {} ,
+  window.OpenOrchestra.DataTable = window.OpenOrchestra.DataTable or {}
```

Comment:
Why are you providing the main and sub namespaces in the function ? You could go with OpenOrchestra and create DataTable inside the function if it doesn't exist, it cleaner and you'd avoid some logic in the main function call.